assets/stylesheets/components/blog-teaser.scss

```diff
@@ -0,0 +1,19 @@
+.blog .content {
```

Comment:
I change that in the `blog-teasers` branch....I was in the flow of things and put `home-page` and  `blog-teasers` in this branch...my bad